src/components/layout/SidebarLinks.tsx

```diff
@@ -24,36 +15,41 @@ export function SidebarLinks() {
   )
 }
 
-const renderLink = (link: SidebarLink, child?: boolean) => (
-  <NavLink
-    label={link.title}
-    p={3}
-    fz="14px"
-    variant="subtle"
-    key={link.to ?? link.title}
-    href={link.to ?? "#!"}
-    classNames={{
-      children: "space-y-1",
-      body: "flex-[2]",
-      section: "flex-[1]",
-    }}
-    renderRoot={(props) => (
-      <Link
-        {...props}
-        className={(active) =>
-          cx(
-            "hover:bg-transparent font-sans",
-            props.className,
-            !child && "sidebar-link-root",
-            child && "space-y-2",
-            child && !active && "sidebar-inactive-child",
-            active && "sidebar-active-child dark-gradient",
-          )
-        }
-      />
-    )}
-    defaultOpened={link.defaultOpened}
-  >
-    {link.children && link.children.map((link) => renderLink(link, true))}
-  </NavLink>
-)
+const renderLink = (link: SidebarLink, child?: boolean) => {
+  const { component: Component } = link
+
+  return (
+    <NavLink
+      label={Component ? <Component /> : link.title}
```

Comment:
Allow rendering the popover menu of "New Question"

Reply:
This isn't the end of the world, but we also render the divider as a link
![image](https://github.com/user-attachments/assets/ac24ade3-fc04-4b4a-8acc-13ef264047dc)

Reply:
@WiNloSt fixed in https://github.com/metabase/shoppy/pull/60/commits/ad523b0ba5c105d688155ab4888ea86d5646705f, thanks!